site/content/docs/user/ingress.md

```diff
@@ -5,11 +5,9 @@ menu:
     parent: "user"
     identifier: "user-ingress"
     weight: 3
+description: |-
```

Comment:
Should this be HTML instead of markdown?

Reply:
it's markdown